splash/resources.py

```diff
@@ -38,7 +39,7 @@ def render(self, request):
             return Resource.render(self, request)
         except BadOption as e:
             request.setResponseCode(400)
-            return str(e) + "\n"
+            return str(e).encode('utf-8') + b"\n"
```

Comment:
Yes, the way to go is to leave out .encode in Python 2.x. If this needs to be done several times you can e.g. add a function like similar to Scrapy's [to_bytes](https://github.com/scrapy/scrapy/blob/936266a61a1ae621e10d48ceb9633a7e245d017b/scrapy/utils/python.py#L110).